CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/PluginException.java

```diff
@@ -70,6 +70,10 @@ public PluginException(PluginNotificationLevel notificationLevel, String message
         this.notificationLevel = notificationLevel;
     }
 
+    public PluginException() {	
+        throw new UnsupportedOperationException("Not supported yet.");
```

Comment:
Was there a plan to implement this constructor? If not, this can probably be removed since there are other constructors (I couldn't see any usage on this one)